spanner/api/SpannerTest/Tests.cs

```diff
@@ -53,6 +57,17 @@ public class SpannerTests
             Command = "Spanner"
         };
 
+        public void Dispose()
+        {
+            try
+            {
+                // Delete database created from running the tests.
+                _spannerCmd.Run("deleteSampleDatabase",

```

Comment:
What happens with the database if this fails?

Reply:
It will be deleted as a beginning step the next time the test is run.
https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/453/files#diff-a502c0e97c603f710fbe44afa5200f6aR86